artifacts/dashboard/karmada-dashboard-api.yaml

```diff
@@ -34,6 +34,9 @@ spec:
           name: karmada-dashboard-api
           image: karmada/karmada-dashboard-api:main
           imagePullPolicy: IfNotPresent
+          env:
+            - name: GIN_MODE
+              value: release
```

Comment:
What's the environment used for? I searched over the code base but didn't find any reference to it.

Reply:
It's the env for gin, more details can refer： https://github.com/gin-gonic/gin/blob/master/mode.go